mk/re.mk

```diff
@@ -392,7 +389,7 @@ endif
 
 endif
 
-ifneq ($(strip $(filter __arm64__ ,$(PREDEF))),)
```

Comment:
__arm64__ maybe is still needed for some platforms (iOS). Should we keep both `ifneq` blocks?

Reply:
__aarch64__ is added additionally.